Clinical-Genomics/scout
# text-break style when comments contain extremely long strings not separated by spaces

Fix #1974. Whenever there are long strings not containing space in comments at the case level or the variant level, the the general report HTML and PDF pages are not looking OK, see #1197 

**How to test**:
1. Make a very long comment of this type **this,is,a,impressively,long,comment,that,shoudnt,exist,in,the,first,place,but,sadly,it,does** at the case level and also at the variant (pinned, causative, dismissed, whichever type goes in the report) level. 
1. Make sure that the general report (HTML and PDF) is not OK using the master branch.
1. Switch to thus branch and see that the report looks fine


**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
